core/src/test/scala/design/hamu/BoardSpec.scala

```diff
@@ -0,0 +1,35 @@
+package design.hamu
+
+import org.scalatest.matchers.must.Matchers
+import org.scalatest.wordspec.AnyWordSpec
+
+class BoardSpec extends AnyWordSpec with Matchers {
+  "Board" should {
+    val b = Board.initFreshBoard
+    "produce a fresh board when evoked" in {
+      b.stringify() must equal("___\n___\n___")
+    }
+
+    val bOneMove = b.move(1, 1, 'X')
+    "respond to a requested move" in {
+      bOneMove.stringify() must equal("___\n_X_\n___")
+    }
+
+    "return a requested space" in {
+      bOneMove.get(1, 1) must equal('X')
+    }
+
+    "respect immutability" in {
+      b.stringify() must equal("___\n___\n___")
+    }
+
+    val bTwoMove = bOneMove.move(3, -1, 'X')
```

Comment:
I'd put this variable inside the test case right below since it's part of the unit test logic.

Reply:
I'm actually not sure how to best do this - since I want the third test case to call from bOneMove, I can't put the variable in the second test case or it would be out of scope.